editor/project_manager.cpp

```diff
@@ -669,77 +646,57 @@ void ProjectDialog::set_mode(Mode p_mode) {
 	mode = p_mode;
 }
 
+void ProjectDialog::set_project_name(const String &p_name) {
+	project_name->set_text(p_name);
+}
+
 void ProjectDialog::set_project_path(const String &p_path) {
 	project_path->set_text(p_path);
 }
 
 void ProjectDialog::ask_for_path_and_show() {
 	// Workaround: for the file selection dialog content to be rendered we need to show its parent dialog.
 	show_dialog();
-	_set_message("");
-
-	_browse_path();
+	_browse_project_path();
 }
 
 void ProjectDialog::show_dialog() {
+	msg->hide(); // TODO: Workaround for https://github.com/godotengine/godot/issues/83546. Also check msg->show() at end of this function.
```

Comment:
https://github.com/godotengine/godot/issues/83546

Reply:
```suggestion
	msg->hide(); // TODO: Workaround for GH-83546. Also check msg->show() at end of this function.
```
This style is used more commonly. Also the comment should be FIXME probably.

Reply:
Changed to:
```cpp
msg->set_custom_minimum_size(Size2(500, 0) * EDSCALE); // HACK: This fixes window size with autowrap label. See: https://github.com/godotengine/godot/issues/80218#issuecomment-1669527304
```

Reply:
Now it is:

```gdscript
msg->set_anchors_and_offsets_preset(Control::LayoutPreset::PRESET_TOP_WIDE); // TODO: Work around enormous popup height. See: https://github.com/godotengine/godot/issues/83546
```